packages/components/src/unit-control/index.js

```diff
@@ -158,6 +158,17 @@ function UnitControl(
 		/>
 	) : null;
 
+	let step = props.step;
+
+	/*
+	 * If no step prop has been passed, lookup the active unit and
+	 * try to get step from `units`, or default to a value of `1`
+	 */
+	if ( ! step ) {
```

Comment:
Per the failing unit tests, the `units` prop may be `false`, so we need:

```suggestion
	if ( ! step && units ) {
```